### DISCUSSION TITLE
[Feature]: Allow passing style on web

### DISCUSSION THREAD

**Discussion by M-i-k-e-l** [TheWidlarzGroup/react-native-video#4990] [Ideas]
### What problem are you trying to solve?

Currently the default style in web is increased to the parent's size. I assume this is to align to the behavior on mobile; however in web it is possible to not send the `style` and allow the video to "auto size" to its real size.

This gives a clean look without any unnecessary margins (or effort).

### Proposed solution

Allow passing `style` on web so the video can "auto size" to its real size instead of being forced to the parent's size.

### Platforms

Web

### Additional context

Possible implementation PR: https://github.com/TheWidlarzGroup/react-native-video/pull/4333

Code sample: See PR

[a comment by an automated account is not shown]

**Comment by TWG-AI** [comment]
Good catch! On web the video currently fills its parent 100% regardless of style, but there's a valid use-case for auto-sizing to the video's natural dimensions. We've noted this — it looks like a great candidate for a community contribution too. Feel free to open a PR if you'd like to tackle it!
